scripts/draft-release.sh

```diff
@@ -15,16 +15,17 @@ fi
 FOLDER="${2-}"
 
 echo "Drafting release"
-github-release release --user storj --repo storj --tag "$TAG" --draft
+current_major_release_version=$(echo "$TAG" | cut -d '.' -f 1-2)
```

Comment:
At least from what i understand, this is parsing vMajor.Minor. Due to this, and having to verify it to be sure, the naming is confusing in my eyes. What about just current_release_version?

Reply:
and we cutting minor part from it here

Reply:
I ran this just earlier and it did cut off the patch part, but minor is still there.

Reply:
You're right, I misunderstood "minor" :)